docs/variants/asus_kgpe_d16/setup.md

```diff
@@ -158,6 +158,22 @@ The script will automatically power off the board and update flash chip
 contents using flashrom. After the operation is finished the routing of SPI
 signals is restored to the platform.
 
+### Preparing the flash chip adapter
+The new flash chip supports more memory and it exists in the SOIC-8 package. 
```

Comment:
This section would need a revamp. It will be pretty hard to read by English natives.